### PR TITLE
AST-1084 - E2E Test case for single post meta font options

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added E2E test case for Single post meta font.

### Screenshots

https://share.getcloudapp.com/eDug2WBl

### How has this been tested?

Run single test - npm run test:e2e:interactive tests/e2e/specs/customizer/blog/single-post/single-post-meta-font.test.js

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/Blog/single-post/single-post-meta-font.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+		await setBrowserViewport( 'medium' );
+		await expect( {
+			selector: '.ast-related-post-content .entry-meta *',
+			property: 'font-size',
+		} ).cssValueToBe(
+			`${ await responsiveFontSize(
+				metaFont[ 'related-posts-meta-font-size' ].tablet,
+			) }${
+				metaFont[ 'related-posts-meta-font-size' ][ 'tablet-unit' ]
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/Blog/single-post/single-post-meta-font.test.js:61]: Please check error on this PR

File: tests/e2e/specs/customizer/Blog/single-post/single-post-meta-font.test.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+				metaFont[ 'related-posts-meta-font-size' ].mobile,
+			) }${
+				metaFont[ 'related-posts-meta-font-size' ][ 'mobile-unit' ]
+			}`,
+		);
+
+		await expect( {
+			selector: '.ast-related-post-content .entry-meta *',
+			property: 'line-height',
+		} ).cssValueToBe( `${ metaFont[ 'related-posts-meta-line-height' ] * metaFont[ 'related-posts-meta-font-size' ].desktop }` + 'px' );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/Blog/single-post/single-post-meta-font.test.js:80]: @SimranChougule1 test case failing at local, please check.

File: tests/e2e/specs/customizer/Blog/single-post/single-post-meta-font.test.js (hunk 1 above)
Comment (bsfsnehal) [reply tests/e2e/specs/customizer/Blog/single-post/single-post-meta-font.test.js]: As discussed, commented code for responsive mode due to GitHub issue.

File: tests/e2e/specs/customizer/Blog/single-post/single-post-meta-font.test.js (hunk 2 above)
Comment (bsfsnehal) [reply tests/e2e/specs/customizer/Blog/single-post/single-post-meta-font.test.js:81]: Working fine

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**